src/index.html

```diff
@@ -5,9 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+
+    <div class="card" data-qa="card">
+      <img
+       src="images/imac.jpeg"
+       alt="imac"
+       class="card__image"
+      >
+      <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__feedback">
+        <div class="card__stars">
+          <div class="card__stars-img"></div>
+          <div class="card__stars-img"></div>
+          <div class="card__stars-img"></div>
+          <div class="card__stars-img"></div>
+          <div class="card__stars-img"></div>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+
+
+      <div class="card__price-container">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price">$2,199</p>
+      </div>
+
+      <button class="card__buy">
+        <a
+         href="#"
+         class="card__buy-link"
+         data-qa="hover"
+         >
+        Buy
+        </a>
```

Comment:
fix indentation according to style guide
```suggestion
        <a
           href="#"
           class="card__buy-link"
           data-qa="hover"
         >
          Buy
        </a>
```